content/03.results-and-discussion.md

```diff
@@ -103,34 +77,25 @@ related taxa - a common challenge in ancient bacteria genome reconstruction
 particular annotations (e.g. genes) of an input reference  is offered by bedtools (@doi:10.1093/bioinformatics/btq033).
 
 New tools supplied with the pipeline include a dedicated 'endogenous DNA'
-calculator (endorS.py, by A.A.V), to give approximate level of
+calculator (endorS.py, by A.A.V), to provide an approximate level of
 on-target DNA yield within a sample. Secondly, ancient metagenomic studies may
 include comparative samples from living day individuals
 (@doi:10.1186/s40168-019-0717-3). To support open data, whilst respecting data
 privacy, nf-core/eager includes a 'strip_fastq' script (by M.B.) which creates
 raw FASTQ files but with reference-genome mapped reads removed. This then
 allows safe upload of sequencing data to public repositories with identifiable
 data removed. When using a human reference genome, nf-core/eager also can give
-estimates of the biological sex of a given individual with Sex.DetEERRmine
+estimates of the biological sex of a given individual with `Sex.DetEERRmine`
```

Comment:
Do we want to do code blocks for all tools? I suspect 99.99% of journals won't allow formatting of this.